app/src/main/java/se/inera/intyg/minaintyg/error/dto/LogErrorRequestDTO.java

```diff
@@ -0,0 +1,15 @@
+package se.inera.intyg.minaintyg.error.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class LogErrorRequestDTO {
+
+  private ErrorDataDTO error;
```

Comment:
Do we need to wrap the error in its own request? Could the members of ErrorDataDTO be part of the LogErrorRequestDTO?